cirq/aqt/aqt_sampler.py

```diff
@@ -0,0 +1,224 @@
+                  repetitions: int = 1,
+                  num_qubits: int = 1) -> List[study.TrialResult]:
+        """Samples from the given Circuit or Schedule.
+
+        In contrast to run, this allows for sweeping over different parameter
+        values.
+
+        Args:
+            program: The circuit or schedule to simulate.
+            Should be generated using AQTSampler.generate_circuit_from_list
+            params: Parameters to run with the program.
+            repetitions: The number of repetitions to simulate.
+            num_qubits: The number of qubits in the system.
+
+        The parameters remote_host and access_token are not used.
+
+        Returns:
+            TrialResult list for this run; one for each possible parameter
+            resolver.
+        """
+        # TODO: Where should we get the num_qubits??
+        # TODO: Probably from the measurement in the circuit!
+        meas_name = 'm'  # TODO: Get measurement name from circuit
+        circuit = (program if isinstance(program, circuits.Circuit) else
+                   program.to_circuit())
+        param_resolvers = study.to_resolvers(params)
+        trial_results = []  # type: List[study.TrialResult]
+        for param_resolver in param_resolvers:
+            id_str = uuid.uuid1()
+            json_list = self._run_api(circuit=circuit,
+                                      param_resolver=param_resolver)
+            results = self._send_json(json_str=json_list,
+                                      id_str=id_str,
+                                      repetitions=repetitions,
+                                      num_qubits=num_qubits)
+            results = results.astype(bool)
+            res_dict = {meas_name: results}
+            trial_results.append(
+                study.TrialResult(params=param_resolver,
+                                  repetitions=repetitions,
+                                  measurements=res_dict))
+        return trial_results
+
+    def run(self,
+            program: Union[circuits.Circuit, schedules.Schedule],
+            param_resolver: 'study.ParamResolverOrSimilarType' = None,
+            repetitions: int = 1,
+            num_qubits: int = 4) -> study.TrialResult:
+        """Samples from the given Circuit or Schedule.
+
+        Args:
+            program: The circuit or schedule to simulate.
+            Should be generated using AQTSampler.generate_circuit_from_list
+            param_resolver: Parameters to run with the program.
+            repetitions: The number of repetitions to simulate.
+            num_qubits: The number of qubits.
+
+        Returns:
+            TrialResult for a run.
+        """
+        return self.run_sweep(program,
+                              study.ParamResolver(param_resolver),
+                              repetitions=repetitions,
+                              num_qubits=num_qubits)[0]
+
+
+class AQTSamplerSim(AQTSampler):
```

Comment:
I was very confused by this class until I realized you were talking about a remote simulator instead of a local simulator. I would rename this to `AQTRemoteSimulator`. It happens to be a sampler. In the future you may have other functionality, so it would be unfortunate to have `Sampler` in the name forever. We also don't use `Sampler` in the name of our simulators; simulator kinda implies sample-ability.

The naming confusion was made worse by the other class called AQTSimulator actually being a local simulator.